### PR TITLE
Adds conditional for bootstrap shim

### PR DESCRIPTION
**Pull request by asuh**
Fix for [issue #95](https://github.com/yeoman/generator-webapp/issues/95)

### CODE REVIEW

File: app/templates/require_main.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ require.config({
     paths: {
         jquery: '../bower_components/jquery/jquery'<% if (compassBootstrap) { %>,
         bootstrap: 'vendor/bootstrap'<% } %>
-    },
-    shim: {
+    }<% if (compassBootstrap) { %> ,
```
Comment (passy) [inline app/templates/require_main.js]: Could you remove the space here?

**Comment by asuh** [comment]
Shoot, I don't know why that gitignore file got into there. Not sure how to revert that out of this. :/

**Comment by addyosmani** [comment]
Update your initial fork/branch to only change the files you intended and force push your changes into the current pull request.

**Comment by asuh** [comment]
I'm going to refork this and submit new pull request because i've royally screwed up things on my end. OS X was creating that .DS_Store file and I tried to ignore it. In other words, I'm still learning how to properly use Git for advanced techniques. Sorry about this.

**Comment by sindresorhus** [comment]
You don't need to create a new PR. You can just force push from a clean local branch to this one.

**Comment by asuh** [comment]
@sindresorhus Please forgive my ignorance. I didn't understand how to force push my clean local branch to the original pull request.

**Comment by sindresorhus** [comment]
You could always Google it, but:

``` bash
git checkout clean-branch
git push origin pr-branch -f
```

Keep in mind you should never force push to branches where others have access, so only to PRs and private remotes.

**Comment by asuh** [comment]
I did search Google for a while but couldn't find any information specifically about Github. When I tried to do a `git push` of the clean branch before, i didn't use this `-f` flag you included nor did i know anything about `clean-branch` or `pr-branch`. Nonetheless, your feedback is very considerate. Thank you.

**Comment by sindresorhus** [comment]
You should probably read up on git ;) https://www.google.no/search?q=git+tutorial&oq=git+tutorial&aqs=chrome..69i57j69i65j0l3j69i64.3684j0&sourceid=chrome&ie=UTF-8

`clean-branch` should be switched out with the name of your clean local branch and `pr-branch` should be switched out with the name of the remote branch your PR is based on. This isn't GitHub specific.

**Pull request closed without merging**